.circleci/config.yml

```diff
@@ -50,9 +74,137 @@ jobs:
+  deploy_superset:
+    machine:
+      image: circleci/classic:201808-01
+    parameters:
+      namespace:
+        description: "Namespace to deploy to"
+        default: "nightly"
+        type: string
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
+      - add_ssh_keys:
+          fingerprints:
+            - "ba:ff:a5:77:9b:87:41:be:09:28:1e:b0:5e:d2:12:5c"
+      - run:
+          name: pull ProjectStrand repo
+          command: |
+            mkdir /ProjectStrand
+            echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' >> ~/.ssh/known_hosts
+            git clone git@github.com:trunghlt/ProjectStrand.git --branch << parameters.branch >> --quiet --single-branch /ProjectStrand
+      - run:
+          name: port-forward deployer
+          background: true
+          command: |
+            echo "$GCR_PASSWORD" | docker login --username _json_key --password-stdin https://us.gcr.io
+            echo "$GCR_PASSWORD" | docker login --username _json_key --password-stdin https://gcr.io
+            env > host.env
+            docker run --rm --env-file=host.env --network=host $GCR_HOSTNAME/$GCR_PROJECT_ID/google-cloud-sdk:latest /bin/bash -c \
+            'echo $GCLOUD_SERVICE_KEY | gcloud auth activate-service-account --key-file=- && \
+            gcloud --quiet config set project $GOOGLE_PROJECT_ID && \
+            gcloud --quiet config set compute/zone $GOOGLE_COMPUTE_ZONE && \
+            gcloud container clusters get-credentials overfit && \
+            kubectl port-forward -n << parameters.namespace >> --address 0.0.0.0 svc/deployer 54321:80'
+      - run:
+          name: Update image
+          command: |
+            echo "$GCR_PASSWORD" | docker login --username _json_key --password-stdin https://us.gcr.io
+            echo "$GCR_PASSWORD" | docker login --username _json_key --password-stdin https://gcr.io
+            for i in $(seq 1 10); do
+              sleep 15
+              docker run --rm --network=host -v /ProjectStrand/internal/pkg/deployerpb/deployer.proto:/proto/deployer.proto \
+              -v /tmp/workspace:/tmp/workspace -w /proto $GCR_HOSTNAME/$GCR_PROJECT_ID/grpc_cli:latest \
+              /grpc_cli call localhost:54321 --protofiles=deployer.proto "overfit.deployer.v1.Deployer.UpdateDeploymentImage" \
+              "image: {repo: '$(cat /tmp/workspace/imagenames/superset.repo)', tag: '$(cat /tmp/workspace/imagenames/superset.tag)'}" && s=0 && break
+              s=$?
+            done
+            exit $s
 workflows:
   version: 2
   test:
     jobs:
       - build_image
+      - build_superset_image
```

Comment:
we already have `build_image` job, why do we need another `build_superset_image` job?

Reply:
has been deleted.